packages/graphql/src/classes/NodeDirective.ts

```diff
@@ -48,29 +48,19 @@ export class NodeDirective {
 
     private mapLabelsWithContext(labels: string[], context: Context): string[] {
         return labels.map((label: string) => {
-            const jwtPath = ContextParser.parseTag(label, "jwt");
-            let ctxPath = ContextParser.parseTag(label, "context");
-
-            if (jwtPath) {
-                ctxPath = `jwt.${jwtPath}`;
-            }
-
-            if (ctxPath) {
-                let mappedLabel = ContextParser.getProperty(ctxPath, context);
-                if (mappedLabel) {
-                    return mappedLabel;
+            if (label.startsWith("$")) {
+                // Trim $context. OR $ off the beginning of the string
+                const path = label.substring(label.startsWith("$context") ? 9 : 1);
+                // Search for the key at the root of the context
+                let labelValue = dotProp.get<string>(context, path);
+                if (!labelValue) {
+                    // Search for the key in cypherParams
+                    labelValue = dotProp.get<string>(context.cypherParams, path);
                 }
```

Comment:
**nitpick** This could be a separate function `searchLabel` making this function a bit easier to follow